glib/src/subclass/signal.rs

```diff
@@ -396,6 +399,13 @@ impl SignalBuilder {
         self
     }
 
+    // rustdoc-stripper-ignore-next
+    /// The signal is returned value type.
+    pub fn return_type_from_type(mut self, type_: impl Into<SignalType>) -> Self {
```

Comment:
```suggestion
    pub fn return_type_from(mut self, type_: impl Into<SignalType>) -> Self {
```
maybe?

Reply:
From what? :)

Reply:
Or maybe return_from_type?

Reply:
Seems inconsistent and you wouldn't find it when searching for `return_type`

Reply:
Except if there could be other `return_type_from_*`, I think `return_type_from` is fine because it reads ok and explicit in expressions such as: `builder.return_type_from(Type::BOOL)`.

Function comment seems wrong btw :) (same for existing function `return_type`).